packages/fiori-elements-writer/src/data/templateAttributes.ts

```diff
@@ -32,7 +32,7 @@ const commonUi5Libs: FrameworkLibs = {
         'sap.ui.generic.app',
         'sap.suite.ui.generic.template'
     ],
-    [OdataVersion.v4]: ['sap.m', 'sap.ushell', 'sap.fe.templates']
+    [OdataVersion.v4]: ['sap.m', 'sap.fe.templates']
```

Comment:
Removing `sap.ushell` breaks `npm run start-local`
It seems we need to ensure `sap.ushell` is not added to the manifest.json libs. 
However I thought it was needed for cross-app navigation?

But I think we need to still add `sap.ushell` in the ui5-local.yaml and in the `flpSandbox.html` `data-sap-ui-libs` parameter.
@tobiasqueck Any thoughts?

Reply:
`sap.ushell` should not be needed. If you run your app in FLP (local or productive) then it should be there. If you don't run it in FLP then the shell related functionality is disabled in `sap.fe`.
So, I would also not think it is required in the `flpSandbox.html` but if you have an example project that fails please share it with me. I am curious.

Reply:
@tobiasqueck  Try 
```
cd packages/fiori-elements-writer/
UX_DEBUG=true UX_DEBUG_FULL=true  npm test
cd test/test-output/lropTemplates/lrop_v4_addtests
npm i && npm run start-local
```
page does not load at http://localhost:8080/test/flpSandbox.html?sap-ui-xx-viewCache=false#lrop_v4_addtests-tile

adding `    - name: sap.ushell` allows the app to load. 
![Screenshot 2024-05-22 at 19 54 21](https://github.com/SAP/open-ux-tools/assets/10102054/0b7643ee-fcf8-4dc6-976b-6d343b1eb2a0)

if I recall adding `sap.ushell` to `data-sap-ui-libs` in flpSandbox.html was for performance change rather than waiting for it to be resolved as a dependency later, go ahead a fetch it early as it will be needed.

Reply:
@devinea I stand corrected! The flp bootstrap does not load the `sap.ushell` lib as I assumed but only creates a global object `sap.ushell`, so, we always need to add `sap.ushell` to the preload libs.
I have corrected it in the `preview-middleware` with 82aa6ccd58d35ede39c5fb2e786f978eee3665a7. Something similar should be done for the `flpSandbox.html` template as well.